homeassistant/components/tradfri/__init__.py

```diff
@@ -70,7 +73,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType):
         load_json, hass.config.path(CONFIG_FILE)
     )
 
-    for host, info in legacy_hosts.items():
+    for host, info in legacy_hosts.items():  # type: ignore
```

Comment:
I think this ignore is ok for now. In a follow up PR we should remove the legacy host parsing. It's been legacy for a long time already.

Reply:
Will make a followup PR later.